text/0706-deprecate-ember-global.md

```diff
@@ -0,0 +1,105 @@
+---
+Stage: Accepted
+Start Date: 2020-14-01
+Release Date: Unreleased
+Release Versions:
+  ember-source: vX.Y.Z
+  ember-data: vX.Y.Z
+Relevant Team(s): Ember.js
+RFC PR: https://github.com/emberjs/rfcs/pull/706
+---
+
+# Deprecate the Ember Global
+
+## Summary
+
+Deprecate the Ember global, `window.Ember`, and replace it fully with
+`import Ember from 'ember';`
+
+## Motivation
+
+The Ember global, available at `window.Ember` (or `globalThis.Ember` in all
+environments) is a mostly legacy API from before Ember adopted native ES
+modules. Modern Ember apps do not use the global, and in general it is mostly
+used by legacy code and occasionally in examples. However, it still exists and
+remains undeprecated.
+
+The primary motivation for deprecating the global object is for _tree shaking_
```

Comment:
@pzuraq maybe I'm missing something, could you please describe how we could do tree shaking if we do `import Ember from 'ember';`? Would build system look which props get used from `Ember` objet imported that way (that *seem* like more complex that tree shaking based on imports)?

Reply:
@SergeAstapov it's actually the opposite, if you do _not_ `import Ember from 'ember';` anywhere in your application, _then_ we can treeshake.

Today, we can't even if you do not import it explicitly, because any code anywhere could reference the Ember global dynamically.

Reply:
@pzuraq thanks for clarification!